packages/monorepo-builder/config/config.php

```diff
@@ -25,6 +25,7 @@
 
     $parameters->set(Option::IS_STAGE_REQUIRED, false);
     $parameters->set(Option::STAGES_TO_ALLOW_EXISTING_TAG, []);
+    $parameters->set(Option::LIMIT_RECENT_TAG_RESOLVING_SCOPE_TO_CURRENT_BRANCH, true);
```

Comment:
The default behavior should not change compatibility.

Reply:
Yeah, leftover 😅. I wanted to left `false` to keep the original behavior. Fixed.